cooltools/api/dotfinder.py

```diff
@@ -31,11 +38,21 @@
 bin2_id_name = "bin2_id"
 
 
-def recommend_kernel_params(binsize):
+def recommend_kernels(binsize):
     """
-    Recommned kernel parameters for the
-    standard convolution kernels, 'donut', etc
-    same as in Rao et al 2014
+    Return a set of convolution kernels that are
+    recommended for dot-calling based on the resolution
+    of the input data.
+
+    At the moment it recommends standard 4-kernels originally
+    defined for the HiCCUPS method: donut, horizontal, vertical,
+    lowerleft
+
+    Kernels are recommended for resolutions near 5, 10
+    and 25 kb at the moment. Typically "dots" are not visible
+    at lower resolutions (>28kb) and data is too sparse for
```

Comment:
most datasets are too sparse

Reply:
maybe there are kernel params we can recommend for 1kb? cc @nvictus @itsameerkat

Reply:
updated text. we'll investigate 1-2kb resolution